test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -375,6 +383,43 @@ describe("GET tests", () => {
     expect(resp.text).toContain(CHECK_YOUR_ANSWERS_PAGE_TRUST_TITLE);
   });
 
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag on`, async () => {
+
+    mockIsActiveFeature.mockReturnValueOnce( false ); // another flag
+    mockIsActiveFeature.mockReturnValueOnce( true ); // FEATURE_FLAG_ENABLE_TRUSTS_WEB flag
+
+    const mockTrust1 = {
```

Comment:
I would have moved the mock data to the related file on the `__mocks__` directory

Reply:
Keep test data separately from test decrease readability of code.
I don't like __mocks__ approach, because there are a lot of mock used only for one/few tests, but connected to all tests. Also is somebody will add/remove/change in __mocks__ it will affect all test, and this is incorrect.

Test should not be independant from other tests/functionality

Reply:
Please do not start adding mocks object on unit tests file, it will became unreadable, I need to see unit tests not mocks data. Something like the [following](https://github.com/companieshouse/overseas-entities-web/blob/aa7d38cd685ad61e085448b7ec5fd398fa518618/test/controllers/trust.details.controller.spec.ts#L39) will make the file unreadable, especially in a complex spec file like this one.

Reply:
In Java we may have a set of base test objects that never change but add a test object for the test in the same file as the test for readability - but I can see that you do not agree with that view.

The way this test is coded I can see all what is going on in the test simply